src/components/@commons-feature/calendar/DailyWorkers.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import FlexContainer from '../../@commons/FlexContainer';
+import { useQuery } from '@tanstack/react-query';
+import { getDailyWorkers } from 'apis/getSchedule';
+import styled from 'styled-components';
+import Text from '../../@commons/Text';
+
+interface Props {
+  date: string;
+}
+
+const DailyWorkers = ({ date }: Props): JSX.Element => {
+  const [y, m, d] = date.split('-').map((e) => Number.parseInt(e));
+  const { data: obj } = useQuery([date], () => getDailyWorkers(y, m, d), { suspense: true });
+
+  return (
+    <FlexContainer $wFull>
+      {obj && (
+        <FlexContainer $direction="row">
+          {obj.data.schedule.map((timeData: any) => (
```

Comment:
여기도 optional chaining으로 축약할 수 있을 것 같네요!
```suggestion
      {<FlexContainer $direction="row">
          {obj?.data.schedule.map((timeData: any) => (
```